WME Form Filler.user.js

```diff
@@ -291,9 +291,7 @@
         var i;
         for (i = 0; i < selection.length; i += 1) {
             if (selection[i].attributes.wazeFeature._wmeObject.hasClosures()) {
-                if (W.model.roadClosures.getByAttributes({
-                    segID: selection[i].attributes.wazeFeature._wmeObject.attributes.id
-                })[0].active) {
+                if (W.model.roadClosures.active) {
```

Comment:
This only checks if the closure layer is active or not, it does not find if the selected segments have an active closure on them. So if a segment has a closure set in the future, it will return true when in the past it's returned false.